cmd/launch.go

```diff
@@ -225,6 +225,8 @@ func launchNonInteractive(h *ec2helper.EC2Helper) {
 		}
 	}
 
+	h.ChangeRegion(simpleConfig.Region)
```

Comment:
This is unrelated to the stated purpose of this PR. Are you sure it is correct and belongs?

Reply:
Necessary for bug that occurs when instance region isnt the same as local session region